src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -115,6 +115,30 @@ else if (config.spanCollector != null) {
     return braveBuilder.build();
   }
 
+  Endpoint buildLocalEndpoint(String serviceName, int port, @Nullable InetAddress configAddress) {
+    Endpoint.Builder builder = Endpoint.builder()
+            .serviceName(serviceName)
+            .port(port);
+    InetAddress address = configAddress != null ? configAddress : getSiteLocalAddress();
+    if (address.getAddress().length == 4) {
+      builder.ipv4(ByteBuffer.wrap(address.getAddress()).getInt());
+    } else if (address.getAddress().length == 16) {
+      builder.ipv6(address.getAddress());
+    }
+    return builder.build();
```

Comment:
Sort of a nitpick, i guess - but there's a couple of branches that aren't covered by tests. Coverage aside, it might be useful as a way of showing how the `ServerConfig` maps to the Brave stuff.